sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts

```diff
@@ -76,7 +80,34 @@ export const FastFrameTemplate = html<FastFrame>`
                             Ultrices nibh nunc vestibulum fames. At lacus nunc lacus eget
                             neque.
                         </p>
-                        Color Pickers go here.
+                        <fast-radio-group name="background color" @change="${(x, c) =>
+                            x.backgroundChangeHandler(c.event as MouseEvent)}>
+                            <label slot="label">Background color</label>
+                            <site-color-swatch value="${x =>
```

Comment:
This is a good opportunity to use `repeat`. You can repeat over `colorPalette` and specify a template that binds to each item in the palette.

```typescript
repeat(x => x.colorPalette, html<string>`
    <site-color-swatch value="${x => x}" backgroundColor="${x => x}"></site-color-swatch>
`)
```